grapple/types/images.py

```diff
@@ -181,6 +197,7 @@ def resolve_src_set(
         info: GraphQLResolveInfo,
         sizes: list[int],
         format: str | None = None,
+        preserve_svg: bool = False,
```

Comment:
Based on a conversation in the Wagtail Slack, I am very much inclined to make `preserve_svg` default to True here (and on the rendition)

ie. until Wagtail (well, Willlow) can actually rasterize SVGs, the only way to work with SVGs is to use `preserve_svg` in Grapple, and `preserve-svg` in regular templates, which adds so much extra effort.

What d'ya think?

Reply:
Yes, I think it very much makes sense. I can't imagine a scenario where you would _not_ want `preserveSvg: true` as a default, even if it were possible to rasterize them.

We need to make sure this doesn't break on Wagtail < 5 though; I remember the code was relying on the fact that, on old Wagtails, it would not be possible to change the default `False` value.

Reply:
I think it should be fine. as `preserve_svg` filters out non-SVG filters - https://github.com/torchbox/wagtail-grapple/blob/d8bb9222b70584aa65b9307bfbb7d6139170fbd8/grapple/types/images.py#L153-L161

Reply:
Yes, but `to_svg_safe_spec` is imported only for Wagtail >= 5; just wanted to mention this, as there don't seem to be a CI matrix that includes older Wagtails: https://github.com/torchbox/wagtail-grapple/blob/d8bb9222b70584aa65b9307bfbb7d6139170fbd8/grapple/types/images.py#L24-L25

---
I can implement the change; do you feel it should be part of this PR or separate?

Reply:
we run tox with tox-gh-actions in CI, so the matrix is in https://github.com/torchbox/wagtail-grapple/blob/main/tox.ini

let's follow up in a separate PR. We'd want to wrap `if instance.is_svg() and preserve_svg: ` in an `if WAGTAIL_VERSION > (5, 0)` to be sure

Reply:
> we run tox with tox-gh-actions in CI, so the matrix is in https://github.com/torchbox/wagtail-grapple/blob/main/tox.ini

@zerolab I think something is missing. I couldn't find a CI job that mentioned an old version of Wagtail so I made a branch to see if I can break it. AFAICT this should have failed CI: https://github.com/mgax/wagtail-grapple/commit/7b52a38c60fbcc4f2cec1bf9554cb82debf320b1